### PR TITLE
Showing US/PST time

### PR DESCRIPTION
**Pull request by yayoiukai**

### CODE REVIEW

File: groups/templates/groups/group_edit.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {% include 'base.html' %}
+{% block content %}
+{% load tz %}
+{% timezone "US/Pacific" %}
```
Comment (dericed) [inline groups/templates/groups/group_edit.html:4]: This is a good start. Could the `timezone` value be set in a config file somewhere.

File: templates/base.html [hunk 2]
```diff
@@ -56,9 +56,11 @@
 </nav>
 
         <div id="content">
-
+        {% load tz %}
+        {% timezone "US/Pacific" %}
```
Comment (dericed) [inline templates/base.html:60]: same comment about user-configurable timezones.

File: groups/templates/groups/group_edit.html (hunk 1 above)
Comment (JeromeMartinez) [reply groups/templates/groups/group_edit.html:4]: and there should be no hardcoded default time zone in the code (e.g. use the one from the server by default).

Comment (yayoiukai) [reply groups/templates/groups/group_edit.html:4]: it is changed to US/Pacific at server side for now. Above code is removed actually...

**Pull request merged**